keps/sig-apps/1828-delete-priority/20210112-delete-priority-annotations.yaml

```diff
@@ -0,0 +1,51 @@
+title: KEP Template
```

Comment:
@drbugfinder-work I created an issue under enhancements: https://github.com/kubernetes/enhancements/issues/2255


@drbugfinder-work @janetkuo as per an offline discussion with Richard, Richard may not be able to work on this KEP for much longer, so how about we merge this as is as provisional for now and I can continue to work on getting it to implementable?

Reply:
@drbugfinder-work to do so, we should complete this file.

Reply:
I think it's fine to leave it here, so we do not lose any history or links to this KEP. I've just moved the KEP to the new README. Can you please check it?

Reply:
The KEP number is an enhancement issue number, not the PR number of the intial KEP. Same thing with the directory name, it should be 2255-pod-cost for example.

Please also update the title as well.

Reply:
please rename this file as well to kep.yaml

Reply:
Ok. I've changed the numbers.